msbuild/Xamarin.MacDev.Tasks.Core/Tasks/XamarinTask.cs

```diff
@@ -152,5 +152,15 @@ protected void FileCopierLogCallback (int min_verbosity, string format, params o
 			}
 			Log.LogMessage (importance, format, arguments);
 		}
+
+		protected string GetNonEmptyStringOrFallback (ITaskItem item, string metadataName, string fallbackValue, string fallbackName = null, bool required = false)
```

Comment:
This is also in https://github.com/xamarin/xamarin-macios/pull/14342/files. I assume one will win and the other will auto resolve away?

Reply:
Yep, that's my hope and intention.